contracts/BondingCurve/BondingCurve.sol

```diff
@@ -180,18 +174,19 @@ contract BondingCurve is Initializable, Ownable {
 
         uint256 tokensToBeneficiary;
         uint256 tokensToDividendHolders;
-        uint256 remainderTokens;
-
-        uint256 dividendPercentage = MAX_PERCENTAGE.sub(_splitOnPay);
 
         tokensToBeneficiary = (amount.mul(_splitOnPay)).div(MAX_PERCENTAGE);
-        tokensToDividendHolders = (amount.mul(dividendPercentage)).div(MAX_PERCENTAGE);
-        remainderTokens = amount.sub(tokensToBeneficiary).sub(tokensToDividendHolders);
+        tokensToDividendHolders = amount.sub(tokensToBeneficiary);
```

Comment:
@tspoff i've simplified this math - looks like the remainder math was not really needed.